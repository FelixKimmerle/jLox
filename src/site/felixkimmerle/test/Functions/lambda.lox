var t = fun (a,b,c) {
    return a+b+c;
};

print t;
print t(1,2,3);

fun takelambda(fn) {
    return fn(1,2,3)+3;
}

print takelambda(t);

print t(1);