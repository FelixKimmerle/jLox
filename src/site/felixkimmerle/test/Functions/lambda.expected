<lambda fn>
6
9
Expected 3 arguments but got 1.
[line 14]
